Gems/LmbrCentral/Code/Source/Builders/LuaBuilder/LuaBuilderWorker.cpp

```diff
@@ -43,6 +57,36 @@ namespace LuaBuilder
                 {
                     AZStd::string watchFolder;
                     AZ::Data::AssetInfo assetInfo;
+// carbonated begin
+#if defined(CARBONATED)
```

Comment:
likewise.

Reply:
Removed the // carbonated begin/end (here and above)